README.md

```diff
@@ -1,20 +1,43 @@
 # \*IPs - Star Improvement Proposals
 *by Zachary Belford, Shane Jonas, Donald McIntyre, Yaz Khoury, Anthony Lusardi, Tomaz Kariz*
 
-Star Improvement Proposals are an open standard for contributors to propose changes, additions, or deprecations of core protocol specifications.
+Star Improvement Proposals is an extension of the BIP/EIP/ECIP process, with some changes to allow for decentralization of the process. Its fundamental change involves determinism of proposal numbers and content-based versioning. 
 
 The Star is meant to be replaced with the prefix/accronym of the users choice, though the prefix MUST be the same for all participants of the particular protocol specification.
 
 For example, All forks of the Star Improvement Proposal relating to the bitcoin network MUST be called `BIP`. The differentiator being the location/domain/github organization to which the \*IP belongs.
 
-Star Improvement Proposals are versioned by the content of their name. That is, in the case of making a proposal, the `version` identifier of the proposal MUST be derived by the last 8 characters of the SHA-3-512 of the proposal name. This `version` naming scheme is deterministic in that groups or individuals may choose to include another's proposal without any numbering conflicts.
+# Changes to BIP/EIP/ECIP
+## Version
+Star Improvement Proposals are versioned by the content of their name. First we create a hash of the preamble, then we use a substring of it to create the version hash.
```

Comment:
This isn't strictly a substring, it's modulo of the hash so something like `hash % (0x10000000000000000)`.

But I'll just change it to read as "subsection" to *hopefully* cut down on confusion.

Reply:
Cool, can be down on a new PR, not sure if it was added to Base58 Branch but I'm resolving this conversation for now.